src/pmpo_fortran.f90

```diff
@@ -51,6 +51,47 @@ subroutine polympo_setMPICommunicator(comm) &
     integer(c_int), value :: comm    
   end subroutine
   !---------------------------------------------------------------------------
+  !> @brief set the MP current element array from a host array
+  !> @param mpmesh(in/out) MPMesh object
+  !> @param numMPs(in) length of array, number of the MPs
+  !> @param array(in) input MP element ID 1D array (numMPs)
+  !---------------------------------------------------------------------------
+  subroutine polympo_setMPCurElmID(mpMesh, numMPs, array) &
+             bind(C, NAME='polympo_setMPCurElmID')
+    use :: iso_c_binding
+    type(c_ptr), value :: mpMesh
+    integer(c_int), value :: numMPs
+    type(c_ptr), intent(in), value :: array
+  end subroutine
+  !---------------------------------------------------------------------------
+  !> @brief get the current element ID MP array from a polympo array
+  !> @param mpmesh(in/out) MPMesh object
+  !> @param numMPs(in) length of array, number of the MPs
+  !> @param array(in/out) output MP element ID 1D array (numMPs), allocated by user
+  !---------------------------------------------------------------------------
+  subroutine polympo_getMPCurElmID(mpMesh, numMPs, array) &
+             bind(C, NAME='polympo_getMPCurElmID')
+    use :: iso_c_binding
+    type(c_ptr), value :: mpMesh
+    integer(c_int), value :: numMPs
+    type(c_ptr), value :: array
+  end subroutine
+  !---------------------------------------------------------------------------
+  !> @brief get the MP positions array from a polympo array
+  !> @param mpmesh(in/out) MPMesh object
+  !> @param nComps(in) degree of Components, should always be 3
+  !> @param numMPs(in) length of the 2nd degree of the array, number of the MPs
```

Comment:
I'd replace 'degree' with 'rank' or 'dimension'.

Reply:
Here is the new comments:
https://github.com/SCOREC/polyMPO/commit/27059d373dd03d77f998f7224de6795c48afc8de#diff-c5f65339dc641fd05805c4e2d5fcd7466edf914ea1c01570a7de725676e2703eL82-R83